### DISCUSSION TITLE
[reset my password][Hw0]

### DISCUSSION THREAD

**Discussion by rabberdabber** [prosyslab-classroom/cs348-information-security#733] [Q&A]
Could I reset my vpn password? I changed it but now I forget my password. Very sorry for my carelessness! my student-id is 20170844

**Comment by sumango3** [comment]
I have changed your VPN account to initial password (which is the most right column of the spreadsheet), and unblocked your account. I hope this solve your problem.

**Comment by rabberdabber** [comment]
I appreciate it so much!
